hosting_docs/source/installation/1-quick-monolith-install.rst

```diff
@@ -40,11 +40,36 @@ SSH into the server with a root or a user with root privileges and do below.
 
 	bash cchq-install.sh install-config.yml
 
+
+Post Installation and Server Administration
+-------------------------------------------
+
+Tracking environements directory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+On succesful installation, the script creates an :ref:`environments config directory<reference/1-commcare-cloud/2-configuration:Creating environments directory>` under `~/environments` that stores all of the configuration. We recommend you to to track this via a version control system such as git, so that you can keep track of the changes and share the directory with other team members so that they can perform server administration using commcare-cloud if required.
```

Comment:
```suggestion
On successful installation, the script creates an :ref:`environments config directory<reference/1-commcare-cloud/2-configuration:Creating environments directory>` under `~/environments` that stores all of the configuration. We recommend you to track this via a version control system such as git, so that you can keep track of the changes and share the directory with other team members so that they can perform server administration using commcare-cloud if required.
```